server/src/sim.rs

```diff
@@ -283,9 +283,9 @@ impl Sim {
         populate_fresh_nodes(&mut self.graph);
 
         let mut fresh_voxel_data = vec![];
-        for fresh_node in fresh_nodes.iter().copied() {
+        for fresh_node in (&fresh_nodes).into_iter() {
```

Comment:
`fresh_nodes` is a `Vec`, not an array, so it's not `Copy`, so this change doesn't improve clarity or concision.